packages/dev-tools/package.json

```diff
@@ -3,14 +3,14 @@
   "version": "0.10.29",
   "main": "build/server/index.js",
   "scripts": {
-    "dev": "NODE_ENV=development ts-node ./server/dev-server",
+    "dev-server": "NODE_ENV=development nodemon --watch server -e js,ts --exec \"ts-node ./server/dev-server\"",
+    "dev-ui": "cd client && expo start --web-only",
     "build": "yarn clean && yarn build-server && yarn extract-fragment-types && yarn build-client",
-    "build-client": "NODE_ENV=production next build && next export -o ./build/client",
+    "build-client": "NODE_ENV=production cd client && expo build:web && mv web-build ../build/client",
```

Comment:
Let's ensure that `web.outputPath` works and use that instead of moving. Also possibly just create a script so we don't depend on expo-cli:
```js
import { Webpack } from '@expo/xdl';
import path from 'path';

async function main(args: any[]) {
  const projectRoot = path.resolve(args[0]);
  console.log('Building', projectRoot);
  try {
    await Webpack.bundleAsync(projectRoot, {
      nonInteractive: true,
      mode: 'production',
    });
    process.exit(0);
  } catch (error) {
    console.log(error);
    process.exit(1);
  }
}

if (require.main === module) {
  main(process.argv.slice(2));
}
```

Reply:
What is `expo.web.build.output`?

Reply:
ah. it's an app.json field

Reply:
```
Cannot delete files/folders outside the current working directory. Can be overridden with the `dangerouslyAllowCleanPatternsOutsideProject` option
```

Reply:
```
Cannot delete files/folders outside the current working directory. Can be overridden with the `dangerouslyAllowCleanPatternsOutsideProject` option
```